### ISSUE TITLE
Preview: "Close All" leaves all instances of OpenConsole.exe running

### ISSUE THREAD

**Issue by vefatica** [microsoft/terminal#14213]
### Windows Terminal version

WindowsTerminalPreview 1.16.2209.21001

### Windows build number

10.0.19044.2130

### Steps to reproduce

Start WT preview.  Open a few tabs, click the "X" and click "Close all".

### Expected Behavior

The instances of OpenConsole.exe are terminated.

### Actual Behavior

All instances of OpenConsole.exe are left running.

Note that this does not happen with the release version of WT (1.15.2209.09004).

**Comment by DHowett** [comment]
Thanks for the report! I believe this is a /duplicate of #14132, where OpenConsole is waiting for teardown to finish before exiting but it's blocked on teardown.
